third_party/docfx/templates/devsite/ManagedReference.common.js

```diff
@@ -33,11 +34,19 @@ exports.transform = function (model) {
       case 'enum':
         model.isClass = true;
         if (model.children) groupChildren(model, classCategory);
-        model[getTypePropertyName(model.type)] = true;
         break;
       default:
         break;
     }
+
+    if (!model.title) {
+      if (model.javaType) {
+        model.title = common.getTitleForTypeProperty(model.javaType, model.name, model.uid)

```

Comment:
Should we use `model.type` instead since if there is `model.javaType` it overwrites it to `model.type`?

Reply:
Yes, thanks! ```model.type``` has been overwritten by ```model.javaType``` at this point, so we don't need an extra branch here.